src/components/inputs/file/file.component.tsx

```diff
@@ -49,6 +49,13 @@ const File: React.FC<FormFieldInputProps> = ({ field, value, setFieldValue }) =>
     },
     [setFieldValue],
   );
+  const [clearedFiles, setClearedFiles] = useState([]);
```

Comment:
Can you move this state declaration at the top? 

Question: Are you using the "clearedFiles" state at any point?

Reply:
noted